test/unit/cmap/connection.test.ts

```diff
@@ -894,6 +895,78 @@ describe('new Connection()', function () {
     });
   });
 
+  describe('supportsOpMsg', function () {
```

Comment:
Just nit-picky here to indicate a floating function. :)

```suggestion
  describe('.supportsOpMsg', function () {
```